web/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/TraceContextCorrelation.java

```diff
@@ -50,8 +50,7 @@ private TraceContextCorrelation() {}
      * @param response
      * @param requestTelemetry
      */
-    public static void resolveCorrelation(HttpServletRequest request, HttpServletResponse response,
-        RequestTelemetry requestTelemetry) {
+    public static void resolveCorrelation(HttpServletRequest request, HttpServletResponse response, RequestTelemetry requestTelemetry) {
```

Comment:
can you minimize formatting changes in this file to help with merge conflicts since it has several changes also in the codeless branch?

Reply:
makes sense

I'd like to set a max width in checkstyle that makes sense for modern monitors. I found this file hard to read.

Reply:
Yeah, i'm not opposed to reformatting in general, just not in this file for now :-)